### PR TITLE
Added new license SCILAB

### LINKED ISSUE #2210: Validate that all licenses (and their URLs) are detected from https://directory.fsf.org/wiki?title=Category:License
There are about ~240 URLs and licenses listed at https://directory.fsf.org/wiki?title=Category:License
We should:
1. ensure that we can detect all of them
2. ensure that we can detect their URL
We should also add as an "other_urls" ech URL to the corresponding license YAML

[fsf-license-urls.txt](https://github.com/nexB/scancode-toolkit/files/5177958/fsf-license-urls.txt)

### PR DESCRIPTION
**Pull request by tushar912**
Signed-off-by: Tushar912 <tushar.912u@gmail.com>

Partially Fixes #2210 

### Tasks

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [x] Commits are in uniquely-named feature branch and has no merge conflicts 📁

### CODE REVIEW

File: src/licensedcode/data/licenses/Scilab-old.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+key: Scilab-old
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/Scilab-old.yml]: The keys and filename are always all lowercase, also is there anything but this old scilab licenses that is a scilab license? If not let's leave aside the old suffix

File: src/licensedcode/data/licenses/Scilab-old.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+short_name: SCILAB
+name: SCILAB License 
+category: Proprietary Free
+owner: INRIA/ENPC
+homepage_url:  https://fedoraproject.org/wiki/Licensing/Scilab License Old
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/Scilab-old.yml]: The URL is incorrect with spaces. It should be :
```suggestion
homepage_url:  https://fedoraproject.org/wiki/Licensing/Scilab_License_Old
```

Bu this is not the homepage_url:  but rather some place other URL.
The homepage was at http://web.archive.org/web/20051212214843/http://www.scilab.org/legal/license.html instead

File: src/licensedcode/data/licenses/Scilab-old.LICENSE [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+9- Applicable law
+
+The present license and its effects are subject to French law and the
+competent French courts.
+
+
+
+
+
+FR:
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/Scilab-old.LICENSE]: We avoid keeping two text in different languages in the same files. These would be two different licenses instead.

**Review by pombredanne — CHANGES_REQUESTED** [review]
Thanks!
See some comments inline

File: src/licensedcode/data/licenses/Scilab-old.yml (hunk 1 above)
Comment (tushar912) [reply src/licensedcode/data/licenses/Scilab-old.yml]: Ok , I think i should rename both key and filename as <code>scilab</code>

File: src/licensedcode/data/licenses/Scilab-old.yml (hunk 2 above)
Comment (tushar912) [reply src/licensedcode/data/licenses/Scilab-old.yml]: Thanks for the homepage_url . I would make the change

File: src/licensedcode/data/licenses/Scilab-old.LICENSE (hunk 3 above)
Comment (tushar912) [reply src/licensedcode/data/licenses/Scilab-old.LICENSE]: Do you mean I should create two licenses for french and english (something like <code>scilab-en</code> and <code> scilab-fr</code>)?

Comment (pombredanne) [reply src/licensedcode/data/licenses/Scilab-old.LICENSE]: yes

File: src/licensedcode/data/licenses/Scilab-old.yml (hunk 1 above)
Comment (pombredanne) [reply src/licensedcode/data/licenses/Scilab-old.yml]: yes

File: src/licensedcode/data/licenses/Scilab-old.yml (hunk 2 above)
Comment (pombredanne) [reply src/licensedcode/data/licenses/Scilab-old.yml]: :+1:

**Review by pombredanne — CHANGES_REQUESTED** [review]

**Comment by pombredanne** [comment]
@tushar912 gentle ping ... now that I pushed a release I would like to merge this soon enough

**Comment by tushar912** [comment]
Sorry @pombredanne for the delay. I would get this ready ASAP.

**Comment by tushar912** [comment]
@pombredanne I have made the changes.

**Pull request merged**